postgres_scanner.cpp

```diff
@@ -397,13 +397,25 @@ static void PostgresInitInternal(ClientContext &context, const PostgresBindData
 		}
 		filter_string = " AND " + StringUtil::Join(filter_entries, " AND ");
 	}
+        if ( task_min == 0 && task_max == POSTGRES_TID_MAX ) {
+          // no use of CTID when task_min / task_max cover the largest range. 
+	  // This includes PostgreSQL compatible database which do not provide CTID (like YugabyteDB)
+          lstate.sql = StringUtil::Format(
+            R"(
+COPY (SELECT %s FROM "%s"."%s" WHERE 1=1                                          %s) TO STDOUT (FORMAT binary);
+)",
+
+            col_names, bind_data->schema_name, bind_data->table_name,                     filter_string);
 
-	lstate.sql = StringUtil::Format(
+	} else {
+          // use of CTID on the task_min / task_max range.		
+	  lstate.sql = StringUtil::Format(
 	    R"(
 COPY (SELECT %s FROM "%s"."%s" WHERE ctid BETWEEN '(%d,0)'::tid AND '(%d,0)'::tid %s) TO STDOUT (FORMAT binary);
```

Comment:
but you will have to add AND back here:
```
COPY (SELECT %s FROM "%s"."%s" WHERE ctid BETWEEN '(%d,0)'::tid AND '(%d,0)'::tid AND %s) TO STDOUT (FORMAT binary);
```

Reply:
We need to add a AND only if there's a filter. I'll add that